### ISSUE TITLE
Distinguishing between "Differentially Private Statistics" and "Other Auxiliary Files" in Dataverse

### ISSUE THREAD

**Issue by pdurbin** [opendifferentialprivacy/opendp-ux#135]
As @ekraffmiller knows since she implemented it, the new API for uploading auxiliary files to Dataverse is quite flexible. You just upload the file and give it format tag and a format version. See for https://guides.dataverse.org/en/5.3/developers/aux-file-support.html for details.

Once these files are in Dataverse, we'd like a way to distinguish between differentially private statistics and other files. Here are two screenshots to illustrate:

## Differentially Private Statistics

<img width="694" alt="Screen Shot 2021-03-26 at 2 43 37 PM" src="https://user-images.githubusercontent.com/21006/112678750-b4118e80-8e41-11eb-8296-7fdd9d53cb09.png">

## Other Auxiliary Files

<img width="687" alt="Screen Shot 2021-03-26 at 2 43 49 PM" src="https://user-images.githubusercontent.com/21006/112678752-b4aa2500-8e41-11eb-9b2e-eef399e95e4f.png">

The rule we have implemented in Dataverse for now (in currently unmerged pull request https://github.com/IQSS/dataverse/pull/7729 ) is that if the format tag starts with "dp" the auxiliary files is put under "Differentially Private Statistics". Otherwise it is put under "Other Auxiliary Files".

I'm not sure what format tags you plan to send to Dataverse so I thought I'd put this on your radar so we can talk about it.

**Comment by TaniaSchlatter** [comment]
It would be a little easier to parse if the convention is "dp_" before the file type. Is this possible?

**Comment by pdurbin** [comment]
@TaniaSchlatter yes, I just added a reminder here: https://github.com/IQSS/dataverse/pull/7729#issuecomment-809439708

**Comment by pdurbin** [comment]
We ended up adding a first-class "type" as of Dataverse 5.5. So, you can send type=DP to upload differentially private statistics. Here's a screenshot of the docs from https://guides.dataverse.org/en/5.5/developers/aux-file-support.html

<img width="874" alt="Screen Shot 2021-05-24 at 1 38 17 PM" src="https://user-images.githubusercontent.com/21006/119386440-84420400-bc95-11eb-9ff0-44e84dc63465.png">

As I mentioned at standup this morning, Dataverse 5.5 has been deployed to https://demo.dataverse.org so we can start playing with this any time.

**Comment by raprasad** [comment]
DP files need to be prominent in the UI (separate tab or something else to be discussed)
